python/sigstore_protobuf_specs/dev/sigstore/bundle/v1.py

```diff
@@ -0,0 +1,63 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# sources: sigstore_bundle.proto
+# plugin: python-betterproto
+from dataclasses import dataclass
+from typing import List
+
+import betterproto
+
+from .dev.sigstore.common import v1
```

Comment:
Unfortunately these relative imports are busted. I'm trying to figure out whether this is a case of PEBCAK or whether there's a bug that needs fixing in `betterproto`.